crates/db_schema/src/schema.rs

```diff
@@ -670,6 +684,7 @@ joinable!(comment -> person_alias_1 (creator_id));
 
 joinable!(post_report -> person_alias_2 (resolver_id));
 joinable!(comment_report -> person_alias_2 (resolver_id));
+joinable!(private_message_report -> person_alias_2 (resolver_id));
```

Comment:
I think this also needs to be joinable to person_alias_1 also? I'm not sure how its working without it.

Reply:
No clue, added it now.